sdk/communication/communication-phone-numbers/test/public/siprouting/utils/recordedClient.ts

```diff
@@ -117,3 +118,18 @@ export async function createRecordedClientWithToken(
   // casting is a workaround to enable min-max testing
   return { client, recorder };
 }
+
+export async function clearSipConfiguration(): Promise<void> {
+  const client = new SipRoutingClient(env.COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING ?? "");
+  await client.setRoutes([]);
+  await client.setTrunks([]);
+}
+
+let fqdnNumber = 1;
+export function getUniqueFqdn(recorder: Recorder): string {
+  const uniqueDomain = uuid().replace(/-/g, "");
```

Comment:
Dashes are replaced from the domain names to avoid the UUID sanitizer in tests, which would break the variables handling.